[MASTER] - Configure loopback callback URIs (#3132)
wso2/docs-is: en/docs/guides/login/oauth-app-config-advanced.md

---

??? note "Click for information on configuring loopback callback URLs"

>      From IS 6.0.0 onwards, to comply with  [RFC 8252 section 7.3](https://datatracker.ietf.org/doc/html/rfc8252#section-7.3), the callback URL in the authorization request does not need to have an exact port match to the callback URL registered here, if it is a loopback callback IP address.

Reviewer: ```suggestion
     From IS 6.0.0 onwards, to comply with [RFC 8252 section 7.3](https://datatracker.ietf.org/doc/html/rfc8252#section-7.3), the callback URL in the authorization request does not need to have an exact port match to the callback URL registered here if it is a loopback callback IP address.
```

>      Only loopback IP addresses (`127.0.0.1` and `[::1]`) are supported for this feature. 

Reviewer: ```suggestion
     Exact port matches are not required if you use loopback IP addresses (`127.0.0.1` and `[::1]`) only. 
```

      For example, if you register a loopback callback URL `http://127.0.0.1:8090/callback` the following callback URLs in the authorization requests would also be valid:
      <ul>
              <li><code>http://127.0.0.1:8090/callback</code></li>
              <li><code>http://127.0.0.1:16000/callback</code></li>
              <li><code>http://127.0.0.1:7500/callback</code></li>
      </ul>
      

>       When registering multiple callback URLs using a regex pattern, do not specify the port number for for the loopback callback URL either as a single port or as a capture group. An example regex pattern is as follows:<br />

Reviewer: ```suggestion
      When registering multiple callback URLs using a regex pattern, do not specify the port number for the loopback callback URL either as a single port or as a capture group. An example regex pattern is as follows:<br />
```
